lib/testing/test_runner.sh

```diff
@@ -69,6 +69,27 @@ OUTSIDE_MAC=
 ENABLE_IPV4=0
 ENABLE_VLAN=0
 
+is_trace_attach_supported()
+{
+    if [[ -z "${TRACE_ATTACH_SUPPORT:-}" ]]; then
```

Comment:
This doesn't actually work since each test is executed in a subshell, so the variable assignment is lost for subsequent runs. Maybe write a file in `$STATEDIR` instead?

Reply:
Done, see the latest change...